dotcom-rendering/src/lib/decidePalette.ts

```diff
@@ -1150,40 +1150,75 @@ const fillShareIconGrayBackground = (format: ArticleFormat): string => {
 };
 
 const fillBlockquoteIcon = (format: ArticleFormat): string => {
-	if (format.design === ArticleDesign.Analysis) {
-		switch (format.theme) {
-			case Pillar.News:
-				return news[300];
-			default:
-				return pillarPalette[format.theme].main;
```

Comment:
Uses [`pillarPalette`](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/lib/pillars.ts#L65) which is deprecated.